packages/xrpl/src/sugar/autofill.ts

```diff
@@ -351,6 +351,7 @@ export async function checkAccountDeleteBlockers(
     account: tx.Account,
     ledger_index: 'validated',
     deletion_blockers_only: true,
+    api_version: client.apiVersion,
```

Comment:
there is one more invocation of `account_info` in this file (line 229). For the sake of consistency, I think it's better to specify `api_version: client.apiVersion` at that line too.

Reply:
and also an invocation to `server_state` request -- but I don't think that request is impacted by API V2 changes

Reply:
Good catch. This line should actually be deleted because `client.apiVersion` is eventually used in `request()`.